test/UserAgentBag-types.test.ts

```diff
@@ -0,0 +1,14 @@
+import { Entry } from '..'
+import test from 'ava'
+import UserAgentBag = require('..')
+
+test('UserAgentBag types', t => {
```

Comment:
Is it worth adding additional tests for `UserAgentBag`'s other methods, like `get` and `getAll`?

Reply:
I originally figured that in the TS test we only need to verify that we can import and instantiate the class in TS, since the JS test already tests the actual logic for each of the methods. But yeah I think it'd make sense to make sure the method types work as well. I've copied over the tests from the JS test file - WDYT?